pkg/registry/admissionregistration/mutatingwebhookconfiguration/storage/storage.go

```diff
@@ -63,3 +63,10 @@ var _ rest.CategoriesProvider = &REST{}
 func (r *REST) Categories() []string {
 	return []string{"api-extensions"}
 }
+
+var _ rest.SingularNameProvider = &REST{}
+
+// SingularName implements the SingularNameProvider interfaces. This returns singular name of core resource.
+func (r *REST) SingularName() string {
+	return "mutatingwebhookconfiguration"
```

Comment:
I think setting a peer field to `DefaultQualifiedResource` above would be clearer (e.g. `SingularQualifiedResource: admissionregistration.Resource("mutatingwebhookconfiguration"),`

(same comment applies to all)

Reply:
Indeed that ^ would be clearer, I'll change the patch this way.